src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -149,7 +149,8 @@ trait BeamHelper extends LazyLogging {
         override def install(): Unit = {
           // This code will be executed 3 times due to this https://github.com/LBNL-UCB-STI/matsim/blob/master/matsim/src/main/java/org/matsim/core/controler/Injector.java#L99:L101
           // createMapBindingsForType is called 3 times. Be careful not to do expensive operations here
-          bind(classOf[BeamConfig]).toInstance(beamConfig)
+          //bind(classOf[BeamConfig]).toInstance(beamConfig)
```

Comment:
Let's just remove this - have you searched for any other injections of beamconfig? It'd probably be caught by the tests if the injection was bad, though.

Reply:
yes @JustinPihony, Have a look into this line, So whenever we inject BeamConfig this will provide BeamConfig Instance https://github.com/LBNL-UCB-STI/beam/blob/7e15d233c88fda573734158c566ee357e9f93ce2/src/main/scala/beam/sim/config/ConfigModule.scala#L14